packages/client/src/TRPCClientError.ts

```diff
@@ -1,10 +1,14 @@
-import type { inferErrorShape, Maybe, TRPCInferrable } from '@trpc/core';
+import type {
+  inferErrorShape,
+  Maybe,
+  TRPCErrorResponse,
+  TRPCInferrable,
+} from '@trpc/server/unstable-core-do-not-import';
 import {
   getCauseFromUnknown,
   isObject,
   type DefaultErrorShape,
```

Comment:
thy is this not getting moved by the linter to the type imports?

Reply:
unsure, but it'll get fixed automatically once prettier updates :D